### DISCUSSION TITLE
<Not getting connected to raw.githubusercontent.com>

### DISCUSSION THREAD

**Discussion by namn-grg** [community/community#42928] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

From past couple of days I am facing a weird problem- I am not getting connected to **raw.githubusercontent.com**. 
2 main points:-
1. None of the image is being seen in the readme even though it is seen from other's pc.
2. I was running this command 'curl -L https://foundry.paradigm.xyz | bash' and this error popped up 'curl: (7) Failed to connect to raw.githubusercontent.com port 443: Connection timed out'. I googled around this a lot and found this [solution](https://bobcares.com/blog/curl-error-7-failed-to-connect-to-port-443/) where they had similar problem and solution was to **contact their team to whitelist the IP in their firewall.** ![image](https://user-images.githubusercontent.com/97289118/210077230-e3229489-ea12-45de-995e-e2fb2544cc48.png)

From these 2 issues I concluded that there is some problem with my laptop address getting connected to **raw.githubusercontent.com** 
Requesting some help please.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @naman-ng ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
